.github/workflows/markdown.yml

```diff
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+name: Auto Format Documentation
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  format:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        # Make sure the actual branch is checked out when running on pull requests
+        ref: ${{ github.head_ref }}
```

Comment:
interesting.. never seen this before. I've never had problems with the action running in the wrong branch without this, but interesting that theres an option to specify.

Reply:
I think it is needed when the action will push changes back to the branch.

Reply:
Ah.. makes sense! cool